media/css/mozorg/home/ijustine-promo.scss

```diff
@@ -0,0 +1,60 @@
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at https://mozilla.org/MPL/2.0/.
+
+@import '~@mozilla-protocol/core/protocol/css/includes/lib';
+
+.t-ijustine {
+    padding: 0;
+
+    .mzp-c-split-bg {
+        background-color: #6f4ae7;
+        padding-top: 0;
+    }
+
+    .mzp-c-split-container {
+        padding: 0;
+    }
+
+    .mzp-c-split-body {
+        .mzp-c-wordmark {
+            display: none;
+        }
+    }
+
+    h2 {
+        @include text-title-md;
+    }
+
+    p {
+        @include text-body-lg;
```

Comment:
I increased the font here so it looks good flowing from a larger header into this sized font. The size before was too small for such a big header. This flows better.